backend/sql/curriculum_scripts/scrape_curriculum.js

```diff
@@ -108,12 +108,19 @@ async function store_curriculum(curriculum) {
         ]
       );
 
-      //Links the courses to the current curriculum
-      await connection.query(
-        "INSERT INTO curriculum_core_classes (curriculum_name,course_code) VALUES(?,?);",
+      let curriculum_core_classes = await connection.query(
+        "SELECT * FROM curriculum_core_classes WHERE curriculum_name = ? AND course_code = ?;",
         [curriculum_name, course.course_code]
       );
 
+      if (curriculum_core_classes.length == 0) {
```

Comment:
Instead of selecting the curriculum_core_classes and checking their length, you can select a count of them and check if that count is zero,since you dont use the courses you select. Ask @Alex for the sql syntax for selecting a count or look it up online.

Reply:
⬆️